### ISSUE TITLE
TouchScript not response for TUIOSimulator

### ISSUE THREAD

**Issue by KevinQ-BIT** [TouchScript/TouchScript#312]
Hi all, I've been using TUIOSimulator from http://wiki.nuigroup.com/TUIOSimulator to simulate TUIO cursors in the development environment. Yet the examples of TouchScript don't response for it. I used visual studio debug tool to insert breakpoints in the function `OnCursorAdded(object sender, TuioCursorEventArgs e)` and the received the information of the cursors, but the touch gestures are not triggered. Also, the settings of TouchScript seems not working, both the 'Enable Debug Mode' and 'Enable TUIO'.

**Comment by KevinQ-BIT** [comment]
Hi all. Here is an update. I double checked both the settings of TouchScript and the Scripting Define Symbols. I'm pretty sure that the ‘TOUCHSCRIPT_TUIO’ has been added to the project already, but it doesn't work. As for the TUIOSimulator, I found that only when starting the project through Visual Studio the gesture will be triggered, neither the play button of Unity Editor nor the released project will do. Is there anything wrong with my configuration of TouchScript? Besides, sorry for asking questions in issues but the link of Official Forum will be redirected to here.
